### PR TITLE
Home work 5.2

### PR DESCRIPTION
**Pull request by graveman13**

### CODE REVIEW

**Comment by Tarabarnik** [comment]
try creating object of Plane (for this purpose create main class and method) and write your code in comment under this

**Comment by graveman13** [comment]
> try creating object of Plane (for this purpose create main class and method) and write your code in comment under this

**Comment by graveman13** [comment]
re-open

**Comment by graveman13** [comment]
> try creating object of Plane (for this purpose create main class and method) and write your code in comment under this
I add new pull re. becouse I did not add static to PlaneBuilder
public class App {
    //First version
    Plane plane1 = new Plane.PlaneBuilder().setHeight(100).setModel("F-5").build();
    ///Second version
    Plane.PlaneBuilder planeBuilder = new Plane.PlaneBuilder().setHeight(51).setModel("UU");
    Plane plane2 = new Plane.PlaneBuilder().build();
}

File: src/main/java/core/basesyntax/Plane.java [hunk 1]
```diff
@@ -7,5 +7,82 @@
...
+        }
+
+        public PlaneBuilder setHeight(int height) {
+            this.height = height;
+            return this;
+        }
+
+        public Plane build() {
+
+            return new Plane(type, model, numberOfSeats, length, height);
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Plane.java]:
```suggestion
            return new Plane(this);
```

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**